tools/github-actions/new-version/readme.md

```diff
@@ -0,0 +1,19 @@
+# New version action
```

Comment:
could reuse the doc of the Azure extension: https://dev.azure.com/AmadeusDigitalAirline/AzureTools/_git/new-version-plugin?path=/README.md